recipes/astromodels/meta.yaml

```diff
@@ -30,6 +30,7 @@ requirements:
     - numba
     - numpy >=1.6
     - omegaconf
+    - antlr4-python3-runtime==4.8
```

Comment:
This is required until https://github.com/conda-forge/omegaconf-feedstock/pull/25 is merged.